apps/util-func/src/adapters/config.ts

```diff
@@ -0,0 +1,8 @@
+import * as z from "zod";
+
+export const envSchema = z.object({
+  COM_STORAGE_ACCOUNT__serviceUri: z.string().url(),
```

Comment:
I think our common resources (`COM` without a specific `app_name`) can be referenced without the prefix
```suggestion
  STORAGE_ACCOUNT__serviceUri: z.string().url(),
```

Reply:
[890a673](https://github.com/pagopa/io-messages/pull/181/commits/890a673982f24123ec68a246975d2593bfb464fa)